src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -35,6 +42,9 @@
     public HelpWindow(Stage root) {
         super(FXML, root);
         helpMessage.setText(HELP_MESSAGE);
+        for (CommandEnum command : CommandEnum.values()) {
+            helpAccordion.getPanes().add(new CommandHelpMessage(command, this).getRoot());
+        }
```

Comment:
i guess we can keep the enum just for this `help` functionality. im not sure if there is a better way, considering that `AddressBookParser.java` simply uses a `switch` statement on `Command.COMAND_WORD`